builtin/credential/okta/path_config.go

```diff
@@ -75,6 +84,8 @@ func (b *backend) pathConfigRead(
 		Data: map[string]interface{}{
 			"Org":     cfg.Org,
```

Comment:
Please use lowercase for these and match the input values, e.g. `ttl` and `max_ttl`. Please also change the existing values -- I know it would be an incompatible change but it's an uncommon endpoint that will be read mostly by humans. It's better for them to match the input.